emcc.py

```diff
@@ -1423,8 +1460,19 @@ def check(input_file):
       if options.separate_asm:
         exit_with_error('cannot --separate-asm when emitting wasm, since not emitting asm.js')
 
-      if '-fsanitize-minimal-runtime' in newargs:
-        shared.Settings.UBSAN_RUNTIME = 1
+      sanitize = set()
+
+      for arg in newargs:
+        if arg.startswith('-fsanitize='):
+          sanitize.update(arg.split('=', 1)[1].split(','))
+        elif arg.startswith('-fno-sanitize='):
+          sanitize.difference_update(arg.split('=', 1)[1].split(','))
+
+      if sanitize & UBSAN_SANITIZERS:
```

Comment:
why does `&` work here?

regardless, I am worried about having that `UBSAN_SANITIZERS` list, and all the duplication of parsing logic using that. Can we link in ubsan based on undefined symbols, like we do other  things?

Reply:
`&` is overloaded as set intersection. As for undefined symbols, I am going to try making that work.